src/utils.ts

```diff
@@ -15,8 +15,11 @@
  *
  */
 
-import { version as pjsonVersion, name as pjsonName } from '../package.json';
+import { name as pjsonName, version as pjsonVersion } from '../package.json';
 import { Attribute } from './models';
+import path from 'path';
+import { TestResp } from './reporter';
```

Comment:
Files such as `utils`,` constants` and others usually don't import stuff from files where they were used. This can lead to circular dependencies.
So please move `TestResp` definition to other place.